lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I'm fine with 80 character comments and 100 character code limits. I'm not sure if it really matters much to adjust the linter for this, but I'd support that if you want to make that change.